src/utils/Firebase/firebase.js

```diff
@@ -101,13 +101,18 @@ class Firebase {
       .get()
       .then(deploymentAccountDoc => {
         const customLessonIds = deploymentAccountDoc.get("customLessons");
-        const customLessonRefs = customLessonIds.map(id =>
+        const customLessonRefs = Object.keys(customLessonIds).map(id =>
```

Comment:
if `customLessonIds` is an obj w the customLessonIds as keys you should rename this variable for clarity